lib/init.coffee

```diff
@@ -53,8 +57,12 @@ module.exports =
       Contents of the argfile are passed to javac as arguments.'
 
 
-  activate: ->
-    require('atom-package-deps').install()
+  activate: (state) ->
+    # state-object as preparation for user-notifications
+    @state = if state then state or {}
+    @state = {}
```

Comment:
@steelbrain: the property is currently being reset to an empty object, to reset any preexisting states. The line will be removed when we use the state object.